eng/scripts/Start-DevOpsRecordings.ps1

```diff
@@ -0,0 +1,26 @@
+[CmdletBinding()]
+param ([Parameter(Mandatory=$True)][int] $PR, [Parameter(Mandatory=$True, ValueFromRemainingArguments=$true)][string[]]$SDKs)
```

Comment:
What are the values of SDKs?  The names of the pipelines?  Should it be `libraryPipelineNames` or something?  I'd suggest including a doc string and maybe linking to our pipelines page so it's easy for folks to go get the correct names.  Alternately, maybe add another script to automate listing the valid pipeline names so it's easy to get the correct value?

Reply:
folder names under `sdk`. storage, iot, core etc.

Reply:
Added docs

Reply:
Nice.  Still worth a link from the test framework README.

Reply:
Yup. Working on it.